### PR TITLE
Cma allocation error

### PR DESCRIPTION
**Pull request by yuriidanilovskyi**

### CODE REVIEW

File: stage2/04-pngview/01-run.sh [hunk 1]
```diff
@@ -8,5 +8,5 @@ install -v -m 644 files/splash_images/*          "${ROOTFS_DIR}/usr/share/splash
 
 on_chroot << EOF
 systemctl enable splashscreen.service
-ln -s /usr/share/splash_images/splash_normal_420.png /opt/splash.png
+ln -sfn /usr/share/splash_images/splash_normal_420.png /opt/splash.png
```
Comment (shtig) [inline stage2/04-pngview/01-run.sh:11]: Does not look related to the subject of the PR, but OK, let it stay here.

**Review by shtig — APPROVED** [review]

**Review by j2ko — APPROVED** [review]
